nucleus/deployment/common/src/main/java/fish/payara/deployment/util/GAVConvertor.java

```diff
@@ -117,7 +120,7 @@
      * the provided GAV as Strings
      */
     private Map<String, String> splitGAV(String GAV) throws MalformedURLException {
-        final String[] splitGAV = GAV.split(",|:");
+        final String[] splitGAV = GAV.split("[,:]");
```

Comment:
Isn't a valid GAV only split by `:`? Doesn't it mean when you find a comma that this is really a list of GAV?

Reply:
According to documentation https://docs.payara.fish/community/docs/5.2020.7/documentation/payara-micro/deploying/deploy-cmd-line.html#deploying-applications-from-a-maven-repository for each GAV needs its own           --deployFromGAV option

Reply:
Yes it is true, but at least uncommon to use a comma nonetheless.

Reply:
Looking back at the commit history around this, comma was the original delimiter as that's the "default" delimiter for Payara CLI (though we do use different delimiters all over the place), and we added in support for using a colon after to be more in line with the usual Maven notation.

So `fish.payara,guppy,1.0` and `fish.payara:guppy:1.0` are both valid, and if you want to deploy more than one artefact you have to specify the option multiple times.